jujugui/static/gui/src/app/components/terminal/terminal.js

```diff
@@ -76,6 +76,9 @@ class Terminal extends React.Component {
         });
         return;
       }
+      if (resp['0'] === 'disconnect') {
```

Comment:
Maybe add a comment explaining that this is sent by terminado when the process it runs exits.

Reply:
Done. Thanks!